### PR TITLE
Support volume snapshot in TF

### PR DESCRIPTION
**Pull request by vishalsawale9**
Adding `VolumeSnapshot` support in the TF provider.

### CODE REVIEW

File: pkg/provider/resource_cluster.go [hunk 1]
```diff
@@ -74,8 +74,8 @@ type ClusterResourceModel struct {
 	PostGIS                    types.Bool                         `tfsdk:"post_gis"`
 	PgBouncer                  *PgBouncerModel                    `tfsdk:"pg_bouncer"`
 	Pause                      types.Bool                         `tfsdk:"pause"`
-
-	Timeouts timeouts.Value `tfsdk:"timeouts"`
+	Timeouts                   timeouts.Value                     `tfsdk:"timeouts"`
+	VolumeSnapshot             types.Bool                         `tfsdk:"volume_snapshot_backup"`
```
Comment (wai-wong-edb) [inline pkg/provider/resource_cluster.go:78]: Have VolumeSnapshot above Timeouts as timeouts as timeouts

**Review by wai-wong-edb — CHANGES_REQUESTED** [review]

File: pkg/provider/resource_cluster.go (hunk 1 above)
Comment (vishalsawale9) [reply pkg/provider/resource_cluster.go:77]: Done. Please check.

**Comment by wai-wong-edb** [comment]
Looks good. But we can merge after I merged v1. After I merge v1 there may need to be some rebasing/merging changes

**Review by edbamolvyavahare — APPROVED** [review]
LGTM

**Review by wai-wong-edb — APPROVED** [review]

**Comment by wai-wong-edb** [comment]
closed. Done in PR https://github.com/EnterpriseDB/terraform-provider-biganimal/pull/570

**Pull request closed without merging**